todotoit/cryptoloji
# Implement URL persistence

This PR implementes a backendless backend using URL as persistent storage.

It encode a hash into the URL for later retrieval.
The obtained URL is the one to share for the end user.
There is no URL shortening up to now.

To test this:

- encrypt a message
- copy the link from share view
- paste it in the browser
- go to decrypt and decrypt message

Everything should work fine.
